src/ontology/mondo-edit.obo

```diff
@@ -61546,6 +61546,8 @@ intersection_of: disease_has_location UBERON:0002137 ! aortic valve
 [Term]
 id: MONDO:0003804
 name: blood protein disease
+comment: Reason: out of scope specific to autosomal disorder. Term to consider: none.
```

Comment:
FYI - I updated this to read Term to consider: none.

I'll fix the rest of these.